### ISSUE TITLE
data loss

### ISSUE THREAD

**Issue by Xuzejian** [policeme/chinese_bert_similarity#6]
hi, i meet this problem:
this line  : saver = self.tf.train.Saver()
Error: DataLossError (see above for traceback): Checksum does not match: stored 2361353507 vs. calculated on the restored bytes 3689058997

maybe because the model file that i download is incomplete. I try to download it again, but still failed.

**Comment by Xuzejian** [comment]
re download model file and it words
